app/src/components/RepoView/Repo.vue

```diff
@@ -183,15 +185,17 @@
 
     methods : {
       handleUp( target ) {
+        if ( this.scriptElements.length < 1 ) {
+          return;
+        }
+        
         let index = [].indexOf.call( this.scriptElements, target );
 
         if ( index === -1 ) {
           index = this.scriptElements.length;
         }
-
-        if ( index > 0 ) {
-          this.scriptElements[ index - 1 ].focus();
-        }
+        
```

Comment:
👍 

for the early escape approach, but I'll revert this parts of this change in master again.

Because 

```
if ( index > 0 ) {
  this.scriptElements[ index - 1 ].focus();
}
```

this is still needed. Currently it's throwing... I missed that one too. 

Logic should be, if the current focused element is not the first one, then focus the one before.